benches/storage/engine/mod.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+mod bench_btree_engine;
+mod bench_rocksdb;
+
+use test::Bencher;
+
+use kvproto::kvrpcpb::Context;
+use tikv::storage::engine::{Engine, Modify, Snapshot};
+
+use test_util::generate_random_kvs;
+use tikv::storage::{Key, CF_DEFAULT};
+
+const KEY_LENGTH: usize = 64;
+const VALUE_LENGTH: usize = 128;
+const ITERATIONS: usize = 1000;
+
+fn fill_engine_with<E: Engine>(
+    engine: &E,
+    target_kvs: &[(Vec<u8>, Vec<u8>)],
+    add_target_kvs_to_engine: bool,
+    expect_engine_key_count: usize,
```

Comment:
s/expect_engine_key_count/engine_target_keys_count

Reply:
let me introduce those args here, 
+ `target_kvs` are the key and values which would be used to test `get()` in the following benchemarks. 
+  `add_target_kvs_to_engine` means whether or not add `target_kvs` to  this engine, if `true`
target_kvs will be inserted in engine, and we can get `target_value`  through `get( target_key)`. Else, those `target_kvs` will not be inserted in engine, following `get(target_key)`will return `None`.
+ `expect_engine_key_count` means the expected total key count in engine after we fill it.

Reply:
I will rename those args and add some comments, it's kind of confused here😂